test/test_ale_toggle.vader

```diff
@@ -74,6 +75,57 @@ Before:
     return l:results
   endfunction
 
+  function! TestVirtualtext(buffer, disabled, current, all)
```

Comment:
Add `delfunction` in `After` so it doesn't leak into other tests.

Reply:
Done.  Thanks for the feedback!